include/s3select.h

```diff
@@ -1191,7 +1193,7 @@ class csv_object : public base_s3object
   bool m_aggr_flow = false; //TODO once per query
   bool m_is_to_aggregate;
   bool m_skip_last_line;
-  size_t m_stream_length;
+  //size_t m_stream_length;
```

Comment:
here (and in other places) just remove unused functions/variable (dont comment). or, if they are needed for clarity reasons, add the [[maybe_unused]] attribute

Reply:
in last PR , several compile-warning flags add to cmake.